index.yaml

```diff
@@ -2507,14 +2507,14 @@ entries:
       catalog.cattle.io/namespace: cattle-logging-system
       catalog.cattle.io/release-name: rancher-logging-crd
     apiVersion: v1
-    created: "2021-06-23T08:47:16.934865-07:00"
+    created: "2021-07-02T20:42:32.097190339Z"
     description: Installs the CRDs for rancher-logging.
-    digest: 24a2952c785a9372e87c04e82d2145601c530d49e7d8dc250a6c53026abb1edb
+    digest: 8543212393b921630119d9f0bbaedd046c713e81c18d125c0f9a0b54083b3281
     name: rancher-logging-crd
     type: application
     urls:
-    - assets/rancher-logging/rancher-logging-crd-100.0.0+up3.10.0.tgz
-    version: 100.0.0+up3.10.0
+    - assets/rancher-logging/rancher-logging-crd-100.0.0+up3.12.0.tgz
+    version: 100.0.0+up3.12.0
```

Comment:
Note that the crd chart is defined in the package/rancher-logging directory, and its version is overridden by the version in `package.yaml`